acme/client.go

```diff
@@ -471,7 +468,7 @@ func (c *Client) requestCertificate(authz []authorizationResource, bundle bool,
 	}
 
 	// TODO: should the CSR be customizable?
```

Comment:
Although unrelated to this branch, to answer the question: yes :smile: It would be useful if the user could provide their own CSR. But again, an issue for another branch/PR...